SRM RN: version 1.2.5 (#3565)
harness/developer-hub: release-notes/service-reliability-management.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest: Version 1.2.5

### New features and enhancements

> Added new filters, Environment-Based and SLO Type, to the SLO listing page. You can now filter the SLO list based on environment or SLO type for improved management. (SRM-15506)

Reviewer: ```suggestion
Added new filters named Environment-Based and SLO Type to the SLO listing page. You can now filter the SLO list on environment or SLO type for improved management. (SRM-15506)
```
(If you wish to use "based on", I would recommend "on the basis of" in this scenario, but even that's not necessary. Just "on" will do.)

### Fixed issues

> - Previously, when any of the simple Service Level Objectives (SLOs) reported NO_DATA for the entire duration, and if the missing data was treated as IGNORE, the composite SLO faced challenges as it necessitated data from all underlying simple SLOs to perform calculations. Consequently, this resulted in the composite SLO failing to produce any records and becoming unresponsive. (SRM-15376)

Reviewer: ```suggestion
- Previously, when any of the simple service-level objectives (SLOs) reported NO_DATA for the entire duration, and the missing data was treated as IGNORE, the composite SLO faced challenges as it required data from all underlying simple SLOs to perform calculations. Consequently, the composite SLO failed to produce any records and became unresponsive. (SRM-15376)
```
> faced challenges 

This is very anthropomorphic. Can we write something else? 

> NO_DATA and IGNORE

Can we replace these strings with words that describe what they represent? 

Would the following construction be an improvement?

> Composite SLOs require data from all underlying simple SLOs to perform calculations. Therefore, previously, if any of the simple service-level objectives (SLOs) failed to report data for the entire duration, and that missing data was ignored, the composite SLO failed to produce any records and became unresponsive. (SRM-15376)

>   This issue has been resolved. Now, when one or more simple SLOs report NO_DATA throughout the duration, even when the missing data is treated as IGNORE, the composite SLO continues to generate records with data for all minutes. Importantly, the error budget for those minutes remains unaffected.

Reviewer: ```suggestion
  This issue has been resolved. Now, if one or more simple SLOs report NO_DATA throughout the duration, even if the missing data is treated as IGNORE, the composite SLO continues to generate records with data for the entire duration. Importantly, the error budget for those minutes remains unaffected.
```
